keyboards/handwired/eagleii/eagleii.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
+	       K0001, K0002, K0003, K0004,               K0007, K0008, K0009, K0010, K0011, \
+	       K0101, K0102, K0103, K0104,               K0107, K0108, K0109, K0110, K0111, \
+	       K0201, K0202, K0203, K0204,               K0207, K0208, K0209,               \
+	       K0301, K0302, K0303, K0304,               K0307, K0308, K0309,               \
+	       K0401, K0402, K0403, K0404,        K0406, K0407,               K0410,        \
+	K0500,                                                                              \
+	       K0601, K0602,        K0604,        K0606,        K0608, K0609, K0610, K0611, \
+	K0700, K0701, K0702,        K0704,        K0706, K0707, K0708, K0709,               \
+	       K0801, K0802, K0803, K0804,               K0807, K0808, K0809, K0810, K0811, \
+	       K0901, K0902, K0903, K0904,               K0907, K0908, K0909,               \
+	K1000,                                                                              \
+	                                   K1105  \
```

Comment:
This part should be rearranged so it's an actual keyboard layout, assuming that is what thi is.

Reply:
If you need any help with this part, this may be a good reference: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/converter/usb_usb/usb_usb.h